### PR TITLE
C955 deploy live logs

### PR DESCRIPTION
**Pull request by BSick7**
This changes the `deploy` and `launch` commands to utilize the server to perform deployments and wait for a healthy deployment. The deployment logs are streamed over a websocket connection.

### CODE REVIEW

File: cmd/create_deploy.go [hunk 1]
```diff
@@ -4,15 +4,16 @@ import (
...
 	client := api.Client{Config: nsConfig}
-	result, err := client.Deploys().Create(appDetails.App.StackId, appDetails.App.Id, appDetails.Env.Id, version)
+	newDeploy, err := client.Deploys().Create(appDetails.App.StackId, appDetails.App.Id, appDetails.Env.Id, version)
 	if err != nil {
-		return fmt.Errorf("error updating app version: %w", err)
-	} else if result == nil {
-		return fmt.Errorf("could not find application environment")
+		return nil, fmt.Errorf("error updating app version: %w", err)
+	} else if newDeploy == nil {
+		return nil, fmt.Errorf("could not find application environment")
```
Comment (ssickles) [inline cmd/create_deploy.go:16]: I don't think this error message is correct, is it? Shouldn't it be something like "unable to create a deploy?"

File: cmd/create_deploy.go [hunk 2]
```diff
@@ -4,15 +4,16 @@ import (
...
-func CreateDeploy(nsConfig api.Config, appDetails app.Details, version string) error {
+func CreateDeploy(nsConfig api.Config, appDetails app.Details, version string) (*types.Deploy, error) {
 	client := api.Client{Config: nsConfig}
-	result, err := client.Deploys().Create(appDetails.App.StackId, appDetails.App.Id, appDetails.Env.Id, version)
+	newDeploy, err := client.Deploys().Create(appDetails.App.StackId, appDetails.App.Id, appDetails.Env.Id, version)
 	if err != nil {
-		return fmt.Errorf("error updating app version: %w", err)
-	} else if result == nil {
-		return fmt.Errorf("could not find application environment")
+		return nil, fmt.Errorf("error updating app version: %w", err)
```
Comment (ssickles) [inline cmd/create_deploy.go:14]: Same with this one, the app version isn't what the user is attempting. They are attempting to deploy so I would expect this to say something like "there was an error creating a deploy"

**Review by ssickles — APPROVED** [review]
Looks good! I just noticed some error logs that seem to be carry overs from some old code. I think we can provide a little better error messages there.

File: cmd/create_deploy.go (hunk 1 above)
Comment (BSick7) [reply cmd/create_deploy.go]: Good catch!

File: cmd/create_deploy.go (hunk 2 above)
Comment (BSick7) [reply cmd/create_deploy.go]: Good catch!

**Review by ssickles — APPROVED** [review]
Seems pretty straight-forward, nicely done!

**Pull request merged**